app/forms/candidate_interface/english_foreign_language/ielts_form.rb

```diff
@@ -41,8 +41,12 @@ def fill(ielts:)
     private
 
       def award_year_is_a_valid_year
+        year_limit = Time.zone.today.year.to_i + 1
+
         if !valid_year?(award_year)
           errors.add(:award_year, :invalid)
+        elsif award_year.to_i >= year_limit
+          errors.add(:award_year, :in_the_future, date: year_limit)
```

Comment:
we're now using this same logic 4 times in different files. What do you think about extracting it into ValidationUtils like we do with valid_year?

Reply:
Not a bad shout lemme have a look 👍